DeceptionGame/Assets/Scripts/MapGeneration/MapController.cs

```diff
@@ -162,12 +162,15 @@ public class MapController : MonoBehaviour
         foreach (MapGenerator.Coord coord in playerLocations) {
             Vector3 spawnLocation = mapGenerator.CoordToWorldPoint(coord);
             PlayerController player = Instantiate(playerPrefab, spawnLocation, Quaternion.identity) as PlayerController;
+            PlayerMovementController mvPlayer = player.gameObject.GetComponent<PlayerMovementController>();
```

Comment:
Maybe just expose it in the PlayerController script in case we want to reference it elsewhere